src/Nncase.Tests/Evaluator/UnitTestEvaluatorMath.cs

```diff
@@ -0,0 +1,1033 @@
+                }
+
+            case UnaryOp.Acosh:
+                {
+                    expect = OrtKI.Acosh(ort);
+                    break;
+                }
+
+            case UnaryOp.Asin:
+                {
+                    expect = OrtKI.Asin(ort);
+                    break;
+                }
+
+            case UnaryOp.Asinh:
+                {
+                    expect = OrtKI.Asinh(ort);
+                    break;
+                }
+
+            case UnaryOp.Cos:
+                {
+                    expect = OrtKI.Cos(ort);
+                    break;
+                }
+
+            case UnaryOp.Cosh:
+                {
+                    expect = OrtKI.Cosh(ort);
+                    break;
+                }
+
+            case UnaryOp.Ceil:
+                {
+                    expect = OrtKI.Ceil(ort);
+                    break;
+                }
+
+            case UnaryOp.Exp:
+                {
+                    expect = OrtKI.Exp(ort);
+                    break;
+                }
+
+            case UnaryOp.Floor:
+                {
+                    expect = OrtKI.Floor(ort);
+                    break;
+                }
+
+            case UnaryOp.Log:
+                {
+                    expect = OrtKI.Log(ort);
+                    break;
+                }
+
+            case UnaryOp.LogicalNot:
+                {
+                    expect = OrtKI.Not(ort);
+                    break;
+                }
+
+            case UnaryOp.Neg:
+                {
+                    expect = OrtKI.Neg(ort);
+                    break;
+                }
+
+            case UnaryOp.Round:
+                {
+                    expect = OrtKI.Round(ort);
+                    break;
+                }
+
+            case UnaryOp.Rsqrt:
+                {
+                    expect = OrtKI.Rsqrt(ort);
+                    break;
+                }
+
+            case UnaryOp.Sign:
+                {
+                    expect = OrtKI.Sign(ort);
+                    break;
+                }
+
+            case UnaryOp.Sin:
+                {
+                    expect = OrtKI.Sin(ort);
+                    break;
+                }
+
+            case UnaryOp.Sinh:
+                {
+                    expect = OrtKI.Sinh(ort);
+                    break;
+                }
+
+            case UnaryOp.Sqrt:
+                {
+                    expect = OrtKI.Sqrt(ort);
+                    break;
+                }
+
+            case UnaryOp.Square:
+                {
+                    expect = OrtKI.Square(ort);
+                    break;
+                }
+
+            case UnaryOp.Tanh:
+                {
+                    expect = OrtKI.Tanh(ort);
+                    break;
+                }
+
+            default:
+                {
+                    throw new ArgumentOutOfRangeException(nameof(op));
+                    break;
+                }
+        }
```

Comment:
It's more elegant to use [switch expression](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression).